python_modules/automation/automation/docker/images/buildkite-test/Dockerfile

```diff
@@ -131,8 +131,8 @@ RUN curl -fsSL https://download.docker.com/linux/debian/gpg | \
     && curl -L "https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose \
     && chmod +x /usr/local/bin/docker-compose
 
-# Install Python build deps
-RUN pip install -U pip setuptools wheel
+# Install uv
+COPY --from=ghcr.io/astral-sh/uv:0.5.4 /uv /bin/
```

Comment:
I wasn't able to use a variable for the version (as I initially tried); I think the issue is related to https://github.com/moby/moby/issues/34482.

This is consistent with how things like the JDK are being installed.

Reply:
So was this just not working before?

Reply:
Previously, the Dockerfile just upgraded pip, and `scripts/install_dev_python_modules.py` installed uv using pip. Since updating `scripts/install_dev_python_modules.py` to just use uv directly (instead of pip to install uv, and then using uv), uv should be installed directly.